activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -408,10 +408,18 @@ def write_multi_entries(entries, expires_in: nil, **options)
           if entries.any?
             if mset_capable? && expires_in.nil?
               failsafe :write_multi_entries do
-                payload = serialize_entries(entries, **options)
-                redis.with do |c|
-                  c.mapped_mset(payload)
+                payloads = serialize_entries(entries, **options)
+                return_value = redis.with do |c|
+                  c.mapped_mset(payloads)
                 end
+
+                if local_cache
```

Comment:
I don't think we can use `local_cache` directly here, see https://github.com/rails/rails/pull/38630#discussion_r387253678.

How about overriding `write_multi_entries` in the LocalCache module to clear the entries' keys?

https://github.com/rails/rails/blob/4b4895e60d3c378c6070f2eebe0a20b607f47080/activesupport/lib/active_support/cache/strategy/local_cache.rb#L128

Reply:
We can, but we'll do the work multiple times in most cases as except for this one special cache for redis, `write_multi_entries` is:

```ruby
        def write_multi_entries(hash, **options)
          hash.each do |key, entry|
            write_entry key, entry, **options
          end
        end
```

But given the restrictions on that specialized `write_multi` (only used if no expiry), I wonder if we wouldn't be better to just remove it.

Reply:
That being said, "the work" is a simple `Hash#[]=` per key. So might not matter much.

Reply:
cc @rafaelfranca what do you think?